arch/x86/include/asm/switcher.h

```diff
@@ -4,9 +4,7 @@
 
 #ifdef CONFIG_X86_64
 #include <asm/processor-flags.h>
-
-#define SWITCH_EXIT_REASONS_SYSCALL		1024
-#define SWITCH_EXIT_REASONS_FAILED_VMETNRY	1025
+#include <uapi/asm/pvm.h>
```

Comment:
Please don't move this code. It is part of switcher API (linux kernel internal API) rather than any ABI.

SWITCH_EXIT_REASONS_SYSCALL is not used for PVM_EXIT_REASONS. (PVM_EXIT_REASONS_SYSCALL and several other code are used for PVM_EXIT_REASONS rather than SWITCH_EXIT_REASONS_SYSCALL nor PVM_SYSCALL_VECTOR)

pvm_get_syscall_exit_reason() should handle SWITCH_EXIT_REASONS_FAILED_VMETNRY explicitly and assign the reason to a code defined in pvm_trace.h. (arch/x86/include/uapi/asm/pvm.h should be renamed to pvm_trace.h). (SWITCH_EXIT_REASONS_FAILED_VMETNRY might be deleted in future)